governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder fromInstance(final Object o) {
+        return fromInstances(o);
+    }
+    
+    public static InjectorBuilder fromInstances(final Object... instances) {
```

Comment:
If we do this, maybe consider renaming the method to something like "bindInstances".